source/nordic_sdk/components/libraries/bootloader_dfu/hci_transport/hci_mem_pool_internal.h

```diff
@@ -42,10 +42,10 @@
 #ifndef MEM_POOL_INTERNAL_H__
 #define MEM_POOL_INTERNAL_H__
 
-#define TX_BUF_SIZE       4u    /**< TX buffer size in bytes. */
-#define RX_BUF_SIZE       32u   /**< RX buffer size in bytes. */
+#define TX_BUF_SIZE       32u    /**< TX buffer size in bytes. */

```

Comment:
please keep the original value. This has to do with how application buffers needed for communication are allocated and maintained by the bootloader during firmware transfer. The original values needed to be changed to these before the bootloader would work for me with the older version of the DFU app. Let's not change this without running this past Nordic.

If you wish to do the proper thing, then please file a ticket against Nordic using their 'mySpace' page (after creating a support account with Nordic) and ask about it.